mysql/errname.go

```diff
@@ -968,6 +968,21 @@ var MySQLErrName = map[uint16]string{
 	ErrColumnStateNonPublic:       "can not use non-public column",
 	ErrIndexStateCantNone:         "index %s can't be in none state",
 	ErrInvalidRecordKey:           "invalid record key",
+	ErrInvalidDDLWorker:           "Invalid DDL worker",
+	ErrUnsupportedDDLOperation:    "Unsupported",
```

Comment:
Unsupported DDL Opearation?

Reply:
This is the common prefix of all UnsupportedDDLOperation errors, for example:

```
errCantDropColWithIndex     = terror.ClassDDL.New(mysql.ErrUnsupportedDDLOperation, mysql.MySQLErrName[mysql.ErrUnsupportedDDLOperation]+" drop column with index")
```

I will add a placeholder at the end to make it read-friendly.